src/participant/prompts/exportToPlayground.ts

```diff
@@ -0,0 +1,32 @@
+import * as vscode from 'vscode';
+
+export class EportToPlaygroundPrompt {
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user build MongoDB queries and aggregation pipelines that perform their task.
+You achieve this by converting user's code written in any proggramming language to the MongoDB Shell syntax.
+Take a user promt as an input string and translate it to the MongoDB Shell language.
```

Comment:
```suggestion
Translate the user's prompt to MongoDB Shell language.
```

Reply:
Sounds good. Let's just fix the typo then:
```suggestion
Take a user prompt as an input string and translate it to the MongoDB Shell language.
```

Reply:
Do you use some extension to check grammar or you just 👀? :)

Reply:
I have this extension installed in my VSCode which helps a bit, just noticed here though.
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker